local/main.go

```diff
@@ -0,0 +1,30 @@
+package main
```

Comment:
got this warning locally

![CleanShot 2023-09-02 at 00 02 32](https://github.com/hookdeck/hookdeck-api-schema/assets/26750127/842fb509-6dc2-42fc-b9e1-083e30ac687a)